docs/source/system_reference_guide/oss_documentation/doi_and_licensing.ipynb

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
<p>This is slightly confusing for me. The first two sentences talk about data, but the rest is software. Is that intentional? I think we'll likely have different guidelines for data given that we won't want data dois to come from Zenodo. We'll be working with ESDIS on the best path forward on data governance in VEDA and can bring that back to the documentation in MAAP.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Oops, forgot to remove "data" from that first sentence. That sounds like a plan though, I think Alex also said we aren't ready just yet to talk about data</p>